holoviews/core/dimension.py

```diff
@@ -1409,10 +1409,16 @@ def options(self, *args, **kwargs):
 
         from ..util import opts
         if options is None:
-            expanded = {}
+            expanded_backends = [({}, backend)]
+        elif isinstance(options, list): # List of Options objects
+            expanded_backends = opts._expand_by_backend(options, backend)
         else:
-            expanded = opts._expand_options(options, backend)
-        return self.opts(expanded, backend=backend, clone=clone)
+            expanded_backends = [(opts._expand_options(options, backend), backend)]
+
+        obj = self
+        for backend, expanded  in expanded_backends:
+            obj = obj.opts(expanded, backend=backend, clone=clone)
```

Comment:
So maybe it's a bit more efficient if we only clone the first time and then set clone=False thereafter.

Reply:
Happy to consider this as an optimization later on if it makes sense. At this time this PR is WIP and a first cut!

Reply:
Sure, seems like a simple optimization, simply set ``clone=False`` on the next line.